conf/config.go

```diff
@@ -71,13 +71,16 @@ import (
 	"github.com/spf13/viper"
 )
 
+// Common constants used for logging and error reporting
 const (
-	noKafkaConfig      = "No Kafka configuration available in Clowder, using default one"
-	noBrokerConfig     = "warning: no broker configurations found in clowder config"
-	noSaslConfig       = "warning: SASL configuration is missing"
-	noTopicMapping     = "warning: no kafka mapping found for topic %s"
-	mappingTopicsError = "warning: an error occurred when applying new topics"
-	noStorage          = "warning: no storage section in Clowder config"
+	filenameAttribute               = "filename"
+	parsingConfigurationFileMessage = "parsing configuration file"
+	noKafkaConfig                   = "No Kafka configuration available in Clowder, using default one"
```

Comment:
error strings should not be capitalized

https://staticcheck.io/docs/checks#ST1005